docs/plugins/examples.rst

```diff
@@ -48,12 +48,11 @@ Enable SAML authentication
     name: saml
     version: 0.1.0
     patches:
-     common-env-features: |
-        "ENABLE_THIRD_PARTY_AUTH" : true
+      common-env-features: |
+        "ENABLE_THIRD_PARTY_AUTH": true
 
-     openedx-lms-common-settings: |
-        # saml special settings
-        THIRD_PARTY_AUTH_BACKENDS = "third_party_auth.saml.SAMLAuthBackend"
+      lms-env: |
+        "THIRD_PARTY_AUTH_BACKENDS": ["third_party_auth.saml.SAMLAuthBackend"]
```

Comment:
Could you please keep the patch in `openedx-lms-common-settings`, and not `lms-env`? When possible, I'd rather avoid adding stuff in the json files. (also, please keep the commend line 55)

Reply:
Thanks for taking a look @regisb! I could change it to something like this...

```yaml
openedx-lms-common-settings: |
  # saml special settings
  AUTHENTICATION_BACKENDS = ["third_party_auth.saml.SAMLAuthBackend", "django.contrib.auth.backends.ModelBackend"]
```

But it feels a bit odd to me to override `AUTHENTICATION_BACKENDS` in this way, since it seems like `THIRD_PARTY_AUTH_BACKENDS` is specifically intended for the purpose of specifying custom auth backends [in edx-platform](third_party_auth.saml.SAMLAuthBackend). I'm new to the world of Open edX though, so I certainly could be misunderstanding something. Let me know if you'd still like me to proceed and I'll make the appropriate changes.

Reply:
Yes, I think that you are right that the patch should be adding entries to the setting, and not override it. Still, I really dislike using the json config file. I think the following solution would be best:

```
openedx-lms-common-settings: |
  # saml special settings
  AUTHENTICATION_BACKENDS += ["third_party_auth.saml.SAMLAuthBackend", "django.contrib.auth.backends.ModelBackend"]
```